### PR TITLE
tyntec SMS Business [update]

### PR DESCRIPTION
**Pull request by ridlees**
---
Please check the following conditions for your PR.

- [x ] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [ x] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

**Comment by dougelliott** [comment]
@ridlees Could you please address the conflicts? Thanks!

**Comment by ridlees** [comment]
Unsure about the Failed test. Names of operation were changed to reflect changes in feedback.

File: certified-connectors/tyntec-SMS/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1,14 +1,15 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "tyntec-SMS",
-    "description": "Send a custom SMS message using the tyntec conversation API. You can also use Status_check to check status (accepted/delivered/failed)",
+    "title": "tyntec SMS Business",
```
Comment (sriyen-msft) [inline certified-connectors/tyntec-SMS/apiDefinition.swagger.json:4]: @ridlees We usually suggest partners to name their connectors with their product or company name, As per your product name, it should be "tyntec SMS", Can you please let us know why you would like to add"Business" to the title.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/tyntec-SMS/apiDefinition.swagger.json (hunk 1 above)
Comment (ridlees) [reply certified-connectors/tyntec-SMS/apiDefinition.swagger.json:4]: Sure, it was a proposed name convention for all new connectors. It should follow the name convention of tyntec products (see https://www.tyntec.com/viber-business-messages, https://www.tyntec.com/whatsapp-business-api, https://www.tyntec.com/sms?country=1893&currency=1820&family=1826)

**Review by dougelliott — APPROVED** [review]

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**